mne/morph.py

```diff
@@ -1142,8 +1141,9 @@ def _apply_morph_data(morph, stc_from):
 
         def _morph_one(stc_one):
             # prepare data to be morphed
-            img_to = _interpolate_data(stc_one, morph, mri_resolution=True,
-                                       mri_space=True)
```

Comment:
In `_apply_morph_data` the arguments to `_interpolate_data` were always True/True...

Reply:
Ahh, I think I see why -- because they get resliced later